projects/angular/src/emphasis/_labels.clarity.scss

```
@@ -168,13 +224,24 @@
       }
 
       &:active {
-        $mySassShadow: label-variables.getLabelColor(gray, bg-color);
-        // OldEdge/IE
-        box-shadow: 0 variables.$clr_baselineRem_1px 0 0 label-variables.getLabelColor(gray, bg-color) inset;
-        box-shadow: 0 variables.$clr_baselineRem_1px 0 0
-          var(mixins.css-variablize-string(label-variables.getLabelColor(gray, bg-color, 'css-var')), $mySassShadow)
-          inset;
-        transform: translateY(0.5px);
+        @include mixins.css-var(
+          background,
+          clr-label-bg-active-color,
+          label-variables.$clr-label-bg-hover-color,
+          variables.$clr-use-custom-properties
+        );
```

Text color mixins are missing for both `&:active` and `&:hover` handlers.

```suggestion
        @include mixins.css-var(
          background,
          clr-label-bg-active-color,
          label-variables.$clr-label-bg-hover-color,
          variables.$clr-use-custom-properties
        );
                @include mixins.css-var(
          color,
          clr-label-active-color,
          label-variables.$clr-label-hover-color,
          variables.$clr-use-custom-properties
        );
```